orangecontrib/text/lda.py

```diff
@@ -100,8 +98,21 @@ def get_topics_table_by_id(self, topic_id):
         return t
 
     def get_top_words_by_id(self, topic_id):
-        topics = self.lda.show_topics(num_topics=-1, num_words=10, formatted=False)
+        topics = self.__topics_words(10)
         if topic_id >= len(topics):
             raise ValueError("Too large topic ID.")
-        return [item[1] for item in topics[topic_id]]
-
+        return topics[topic_id]
+
+    def __topics_words(self, num_of_words):
```

Comment:
Two underscores in Python signifies a _private_ member, visible only by instances of this _exact_ class but no subclasses. When classes are not really meant to be extended, and the possibility of overriding these private methods is not taken into account or well thought through, I find such usage ambiguous and even dangerous.

This code implies, first, that the `LDA` class is potentially meant to be extended by subclasses, and second, that the `__topics_words` method needs to be exactly this. So if functionality ever needs to be overriden, wherever this method is _used_, that whole other method (this method's caller) has to be overriden (as opposed to only this method if it were started with a single underscore (`_topics_words`) and be marked _protected_.

The top-voted answers here are relevant: http://stackoverflow.com/questions/797771/python-protected-attributes
I'm sure there's more wisdom to be found on why avoid "private private" members in Python, C++, Java, or altogether.